pkg/storkctl/common_test.go

```diff
@@ -61,7 +62,13 @@ func resetTest() {
 	fakeOCPClient := fakeocpclient.NewSimpleClientset()
 	fakeOCPSecurityClient := fakeocpsecurityclient.NewSimpleClientset()
 	fakeOCPConfigClient := fakeocpconfigclient.NewSimpleClientset()
-	fakeDynamicClient := fakedynamicclient.NewSimpleDynamicClientWithCustomListKinds(scheme, appregistration.GetSupportedGVR())
+	gvrToListKind := appregistration.GetSupportedGVR()
+	gvrToListKind[schema.GroupVersionResource{Group: "apps", Version: "v1", Resource: "deployments"}] = "DeploymentList"
```

Comment:
Do we have to add the custom resource kinds in each reset?

Reply:
There is another unit test TestActivateDeactivateMigrations which uses this function and as a result fakeDynamicClient. So the fakeDynamicClient needs to be initialised with api-resources as well. Earlier implementation only initialised it with resource kinds from application registration.